### ISSUE TITLE
SBOL Blogs

### ISSUE THREAD

**Issue by PrashantVaidyanathan** [SynBioDex/SbolStandardWebsite#245]
We need a page to talk about the planned SBOL Blog. We can add a coming soon text with a small blurb telling people what to expect. Here are a few ideas/examples:

We need to finalise a name for this. Here are a few ideas:
- SBOL Blog
- SBOL Press
- SBOL Insights
- The SBOL Standard Posts
- SBOL Pulse
(open to any other idea too)

**Comment by PrashantVaidyanathan** [comment]
@CarolusVitalis @tcmitchell @LukasBuecherl @GeorgieHauSorensen @Gonza10V

**Comment by CarolusVitalis** [comment]
I agree with Tom; I think that better encompasses what we are aiming for with the blog
